### PR TITLE
Bt flake instancecreate

### PR DESCRIPTION
**Pull request by billyjacobson**
Fixes #issue

> It's a good idea to open an issue first for discussion.

- [ ] I have followed [Sample Format Guide](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/SAMPLE_FORMAT.md)
- [ ] `pom.xml` parent set to latest `shared-configuration`
- [ ] Appropriate changes to README are included in PR
- [ ] API's need to be enabled to test (tell us)
- [ ] Environment Variables need to be set (ask us to set them)
- [ ] **Tests** pass:   `mvn clean verify` **required**
- [ ] **Lint**  passes: `mvn -P lint checkstyle:check` **required**
- [ ] **Static Analysis**:  `mvn -P lint clean compile pmd:cpd-check spotbugs:check` **advisory only**
- [ ] Please **merge** this PR for me once it is approved.

### CODE REVIEW

File: bigtable/src/test/java/com/m/examples/bigtable/QuickstartTest.java [hunk 1]
```diff
@@ -56,6 +58,11 @@ public void setUp() {
     System.setOut(new PrintStream(bout));
   }
 
+  @After
+  public void tearDown() {
+    System.setOut(oldStdout);
```
Comment (averikitsch) [inline bigtable/src/test/java/com/m/examples/bigtable/QuickstartTest.java:63]:
```suggestion
    System.setOut(null);
    bout.reset();
```
I am more use to seeing something like this.

File: bigtable/src/test/java/com/m/examples/bigtable/InstanceAdminExampleTest.java [hunk 2]
```diff
@@ -91,18 +94,22 @@ public void after() {
   }
 
   @Test
-  public void testCreateAndDeleteInstance() throws IOException {
+  public void testCreateAndDeleteInstance() {
```
Comment (averikitsch) [inline bigtable/src/test/java/com/m/examples/bigtable/InstanceAdminExampleTest.java:97]: Usually we let the test throw the exception.

**Comment by lesv** [comment]
It's still not showing up in the logs.  I also looked in Sisyphus. 
# Java 8
```
------------------------------------------------------------
- testing bigtable
------------------------------------------------------------
[ERROR] Tests run: 4, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 81.323 s <<< FAILURE! - in com.m.examples.bigtable.InstanceAdminExampleTest
[ERROR] com.m.examples.bigtable.InstanceAdminExampleTest.testGetInstance  Time elapsed: 2.767 s  <<< FAILURE!
java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:87)
	at org.junit.Assert.assertTrue(Assert.java:42)
	at org.junit.Assert.assertNotNull(Assert.java:713)
	at org.junit.Assert.assertNotNull(Assert.java:723)
	at com.m.examples.bigtable.InstanceAdminExampleTest.testGetInstance(InstanceAdminExampleTest.java:119)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:365)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:273)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:159)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:377)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:138)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:465)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:451)

[ERROR] Failures:
[ERROR]   InstanceAdminExampleTest.testGetInstance:119
```

**Comment by lesv** [comment]
https://stackoverflow.com/questions/8180315/output-failed-test-details-to-stdout-using-maven-surefire

Try adding to your `<properties>` section:
`<maven.test.redirectTestOutputToFile>true</maven.test.redirectTestOutputToFile>`

**Comment by billyjacobson** [comment]
@lesv Should I be doing anything else with this or just wait to hear back about those issues with printing to stdout?

**Comment by lesv** [comment]
I'll review and LGTM, feel free to merge, I'll fix repo tools, but it takes a bit and I want to finish what I'm working on as well.  I'd move to repo tools if this was failing.

**Comment by lesv** [comment]
This appears to have been abandoned.

**Pull request closed without merging**